### PR TITLE
Macos rebranding

### PR DESCRIPTION
**Pull request by Bioblaze**
I need to finish the MacOS side of the CI/CD and the naming for launching the templates, etc etc are needed to be done a specific way so now that i found this need to modify it.

Can peeps help look it over?

### CODE REVIEW

**Comment by Ughuuu** [comment]
Looks good, but this will anyway build a .universal file. I wonder how godot does when exporting a build, as they have a .app file in the end. The .app file opens when double clicking it, while the .universal file you need to run from terminal to start it.

**Review by RandomOfNoWhere — APPROVED** [review]

File: misc/dist/macos/editor_info_plist.template [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
+	<string>Blazium</string>
 	<key>CFBundleName</key>
-	<string>Godot</string>
+	<string>Blazium</string>
 	<key>CFBundleIconFile</key>
-	<string>Godot.icns</string>
+	<string>Blazium.icns</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.godotengine.godot</string>
+	<string>org.blaziumengine.blazium</string>
```
Comment (jss2a98aj) [inline misc/dist/macos/editor_info_plist.template:14]: Should this be `app.blaziumengine.blazium` if going by domain name derived conventions?

File: misc/dist/macos_tools.app/Contents/Info.plist [hunk 2]
```diff
@@ -5,21 +5,21 @@
...
+	<string>Blazium</string>
 	<key>CFBundleName</key>
-	<string>Godot</string>
+	<string>Blazium</string>
 	<key>CFBundleIconFile</key>
-	<string>Godot.icns</string>
+	<string>Blazium.icns</string>
 	<key>CFBundleIdentifier</key>
-	<string>org.godotengine.godot</string>
+	<string>org.blaziumengine.blazium</string>
```
Comment (jss2a98aj) [inline misc/dist/macos_tools.app/Contents/Info.plist:14]: Same with this, if applicable.

**Review by jss2a98aj — APPROVED** [review]

File: misc/dist/macos/editor_info_plist.template (hunk 1 above)
Comment (Bioblaze) [reply misc/dist/macos/editor_info_plist.template:14]: See idk... :| I'm waiting to get access to apple, to see if I set this value there, and shit. I relaly don't know @.@

File: misc/dist/macos_tools.app/Contents/Info.plist (hunk 2 above)
Comment (Bioblaze) [reply misc/dist/macos_tools.app/Contents/Info.plist:14]: same with the statement above

**Review by Norrox — APPROVED** [review]

File: misc/dist/macos/editor_info_plist.template (hunk 1 above)
Comment (Starkium) [reply misc/dist/macos/editor_info_plist.template:14]: isn't this similar to how android works where it goes com.{studio name}.{project name}?
idk about mac, but my guess is that it's just their package convention

**Review by Starkium — APPROVED** [review]
org or not, doesn't bother me, the other changes look good so we can just come back to this later if need be

File: misc/dist/macos/editor_info_plist.template (hunk 1 above)
Comment (jss2a98aj) [reply misc/dist/macos/editor_info_plist.template:14]: After messing with some stuff on the Android side, I think it should be changed to use `app.blaziumengine`

**Pull request closed without merging**